portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,31 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomBibleVerse() {
+  const bibleVersesText =
```

Comment:
Right now, every time a user clicks the button, these arrays are being built. It would be better to construct these outside the function as a constant (see https://google.github.io/styleguide/jsguide.html#naming-constant-names for naming conventions), so they are built once and referenced in this function every time the user clicks the button.

Reply:
Done!